signup.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/reset-css@4.0.1/reset.min.css"
+    />
+    <link rel="stylesheet" href="signup.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+</head>
+<body>
+    
+    <!-- 전체 컨테이너 -->
+    <form>
+        <header>
+            <a href="/">
+                <img src="images/logo.png" alt="판다마켓" class="header__logo">
+            </a>
+        </header>
+
+       <!-- 아이디 비밀번호 박스 -->
+       <div class="container">
+           <div class="email-box">
+               <label class="email-box__text">이메일</label>
+               <input class="email" placeholder="   이메일을 입력해주세요" type="email" onfocus="this.placeholder = ''" />
+           </div>
+           <div class="nickname-box">
+               <label class="nickname-box__text">닉네임</label>
+               <input class="nickname" placeholder="   닉네임을 입력해주세요" type="email" onfocus="this.placeholder = ''" />
+           </div>
+           <div class="password-box">
+               <label class="password-box__text">비밀번호</label>
+               <input class="password" placeholder="   비밀번호를 입력해주세요" type="password" onfocus="this.placeholder = ''" /> 
+               <img src="images/closedEye.png" alt="비밀번호감긴눈" class="password-eye">
+            </div>
+            <div class="passcheck-box">
+                <label class="passcheck-box__text">비밀번호 확인</label>
+                <input class="passcheck" placeholder="   비밀번호 다시 한 번 입력해주세요" type="email" onfocus="this.placeholder = ''" />
```

Comment:
```suggestion
                <input class="passcheck" placeholder="   비밀번호 다시 한 번 입력해주세요" type="password" onfocus="this.placeholder = ''" />
```
type이 잘못 들어가 있는것 같아요!